README.md

```diff
@@ -0,0 +1,219 @@
+
+Generally, the easiest way to get in touch with me. I try to stay on top of Slack and respond in less than an hour during the workday. The one complication is that I'm frequently in meetings. It's my goal to be focused in a meeting and not doing other things. I typically play catch up on Slack between meetings. In the event that I'll need more time to respond, I will try to let you know that.
+
+### Email
+
+I'm quite averse to it and try to spend as little time as possible dealing with it. Typically, I'll lump my email processing into a morning session and an afternoon session. However, if the day is full, this can slip. When something is urgent, please do not use email or we're both going to have a sad time.
+
+### PagerDuty
+
+If it's off-hours and urgent, please page me.
+
+### Mobile phone
+
+I generally try to avoid disseminating my mobile number at work. If you think you'll need it, let's talk.
+
+### Social media
+Nope.
+
+## Bugs & Quirks
+
+Like most humans, I come equipped with a handful of flaws and personality traits that you may find challenging. There are ways in which I try to control or compensate for each of these tendencies but I'm not always successful. My hope is that by being open about these aspects of my personality you will feel comfortable calling them out if they become a problem for you.
+
+### Direct and intense
+
+My [Insights personality profile](https://github.com/matthewroot/root-readme/blob/master/matthew_root_insights_insights_color_breakdown.pdf) is _quite_ red. When we interact, I probably won't beat around the bush and may be more direct than you prefer. I also tend to become more intense as I bring more of my focus to bear on a problem. My brow may furrow and I might even look mad. Be aware this is typically not the case, it's just how I look when my brain is getting down to business on a good challenge. If my directness or intensity becomes too much, there's no harm in letting me know so I can dial it back a bit.
+
+### Fast talker, sporadic rambler
+
+This manifests most acutely when I'm excited about a topic. If I'm ever talking so fast that you can't understand me, kindly ask me to slow down and//or repeat anything you may have missed. When I get on a ramble, I'll typically call it out on my own and take stock with you about whether what I'm saying is useful. If that doesn't happen, please don't feel bad about interrupting me and steering back to the point.
+
+### Stubborn
+
+My least favorite bug. When I've got a strong opinion, my tendency is to dig in and defend it. This is most problematic if I transition away from being open-minded in a discussion. However, I love a good argument and am always pleased to have my mind changed as it usually means I'm seeing the world with more truth. If you find me pushing back against you, lean into it and help me to hear you. If you ever find me being overly stubborn, point it out; it won't be a surprise although you may catch me getting red from embarrassment.
```

Comment:
> I love a good argument

Would this still be accurate and more amicably phrased as "_I love a good **debate**_"?

Reply:
Yes, that is definitely a better way to phrase it. Thank you. :)